internal/player/player.go

```
@@ -50,6 +67,14 @@ type ipcReply struct {
 	RequestID int             `json:"request_id"`
 }
 
+func (p *mpvPlayer) killStartedProcess() {
+	if p.cmd != nil && p.cmd.Process != nil {
+		p.cmd.Process.Kill()
```

golangci-lint (2.12.2)

[error] Error return value of `p.cmd.Process.Kill` is not checked

(errcheck)